corehq/apps/app_manager/static/app_manager/js/preview_app.js

```diff
@@ -38,7 +39,8 @@ hqDefine('app_manager/js/preview_app.js', function() {
     };
 
     _private.navigateBack = function() {
-        var previewWindow = $appPreviewIframe[0].contentWindow;
+        var $appPreviewIframe = $(module.SELECTORS.PREVIEW_WINDOW.IFRAME),
+            previewWindow = $appPreviewIframe[0].contentWindow;
```

Comment:
this fixes a minor issue in this PR: https://github.com/dimagi/commcare-hq/pull/13774#discussion_r85836487